api/src/services/connected-user-log.js

```diff
@@ -0,0 +1,47 @@
+const moment = require('moment');
+const db = require('../db');
+
+const LOG_TYPE = 'connected-user-';
+const TIME_INTERVAL = 30 * 60 * 1000; // 30 minutes
+
+const saveLog = (log) => {
+  if (!log || !log.user) {
+    const error = new Error('Error when saving log: Log Information missing');
+    return Promise.reject(error);
+  }
+  const id = LOG_TYPE + log.user;
+  return db.medicLogs
+    .get(id)
+    .catch(error => {
+      if (error.status === 404) {
+        return { _id: id };
+      }
+      throw error;
+    })
+    .then(doc => {
+      if (log.timestamp - doc.timestamp < TIME_INTERVAL) {
+        return;
+      }
+      const logDoc = Object.assign(doc, log);
+      return db.medicLogs.put(logDoc);
+    });
+};
+
+const getLogs = (interval) => {
```

Comment:
Perhaps I've just thought about this for too long, but I don't think `interval` is the correct name here. `daysAgo` seems more appropriate.